File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
def min_max(n):
    min=1000001
    max=-1000001
    num=0
    L=input().split()
    for i in range(0,n):
        num=int(L[i])
        if num<min:
            min=num
        if num>max:
            max=num
    print(min,max)

n=int(input())
L=[]
min_max(n)

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.*;
import java.util.*;
public class Main {

	public static void main(String[] args) throws IOException {
			
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		StringTokenizer st;
		Integer.parseInt(br.readLine());
        st = new StringTokenizer(br.readLine());
        
        int max = -1000001;
		int min = 1000001;
		
		while(st.hasMoreTokens()) {
			int num = Integer.parseInt(st.nextToken());
			if (num > max) {
				max = num;
			} 
            if (num < min) {
				min = num;
			}
		}
		bw.write(min+" "+max);
		bw.flush();
	}
}